src/lib/leaflet-utils/layer-factory.js

```diff
@@ -0,0 +1,126 @@
+import L from 'leaflet'
+
+import 'leaflet.markercluster'
+import 'leaflet.markercluster/dist/MarkerCluster.css'
+import 'leaflet.markercluster/dist/MarkerCluster.Default.css'
+
+import mapConfig from '../../map.config.js'
+import { redIcon, blackIcon, greenIcon } from './markers'
+
+import './cluster-icon.css'
+
+const BREACH_PRIMARY = 'geo_doorbraaklocaties_primair'
+const BREACH_REGIONAL = 'geo_doorbraaklocaties_regionaal'
+const BREACH_OUTSIDE_DIKE = 'geo_scenariolocaties_buitendijks'
+const BREACH_FLOODING = 'geo_scenariolocaties_wateroverlast'
+const BREACHES_IDS = [
+  BREACH_PRIMARY,
+  BREACH_REGIONAL,
+  BREACH_OUTSIDE_DIKE,
+  BREACH_FLOODING
+]
+const DEFAULT_ICON = new L.Icon.Default()
+const DYNAMIC_GEOSERVER_URL = mapConfig.services.DYNAMIC_GEOSERVER_URL
+const STATIC_GEOSERVER_URL = mapConfig.services.STATIC_GEOSERVER_URL
+
+export default function renderLayer (layer, { breachCallBack }) {
```

Comment:
render?  or create? It's not rendering is it?  
There is rendering in Leaflet, but this has nothing to do with it 
https://leafletjs.com/reference-1.3.4.html#renderer